scenarioo-client/app/manage/builds/build-list.component.ts

```diff
@@ -0,0 +1,81 @@
+import {Component, OnInit} from '@angular/core';
+import {tap} from 'rxjs/operators';
+import {BuildImportStatesResource} from '../../shared/services/buildImportStatesResource.service';
+import {IBuildImportSummary} from '../../generated-types/backend-types';
+import {ConfigurationService} from '../../services/configuration.service';
+import {BuildReimportResource} from '../../shared/services/buildReimportResource.service';
+import {BuildImportService} from '../../shared/services/buildImport.service';
+
+@Component({
+    selector: 'sc-build-list',
+    template: require('./build-list.component.html'),
+})
+export class BuildListComponent implements OnInit {
+    buildImportStates: IBuildImportSummary[];
```

Comment:
initialize with empty array

Reply:
better store an observable and use the async pipe

Reply:
done